# TrajecotryService.srv
float64[7] target_pq
---
string response